integration-tests/test_client_options.py

```diff
@@ -87,3 +88,26 @@ def test_support(insights_client):
     assert "Running command:" in support_result.stdout
     assert "Process output:" in support_result.stdout
     assert "Support information collected in" in support_result.stdout
+
+
+def test_check_show_results(insights_client):
+    """
+    Verify a remediation is returned when checking results from server (--check-results)
+    and then displaying then (--show-results)
+    First I will intorudce a vulnerability to the system and at the
+    end of the test I will set the correct chnod again
+    """
+    insights_client.register()
+    assert conftest.loop_until(lambda: insights_client.is_registered)
+
+    os.chmod("/etc/ssh/sshd_config", 0o777)
```

Comment:
note that the changes to this file must be revert at the end of the test; what you can do is wrap the code of the test in a `try ... finally:` block, and in `finally` revert the file changes: this way, errors are propagated, and the file is restored in any case

Reply:
The change to the file was reverted with the `os.chmod("/etc/ssh/sshd_config", 0o600)` where I set the file permission back to the state it should be, although I edited it to be in a try - finally block.